src/MainAppRouter.js

```diff
@@ -92,6 +90,11 @@ export default function MainAppRouter({ api, setUser, hasExtension }) {
         render={() => <ExtensionInstalled api={api} />}
       />
 
+      <Route
```

Comment:
I mentioned this in our call, but I believe this needs to be set to use PrivateRoute rather than just route, to ensure you have a user login. If not, the page will render with no tags.

Reply:
Yes, I fully agree. It uses Private Route now.